Cahn-Hilliard documentation (#793)
lethe-cfd/lethe: doc/source/parameters/cfd/initial_conditions.rst

   subsection temperature
     set Function expression = 0
   end
   
   subsection cahn hilliard

>      set Function expression = if (x<0.5 & y<1, 1, 0); 0

Reviewer: ```suggestion
     set Function expression = if (x<0.5 & y<1, 1, -1); 0
```
It has to be defined at -1 and 1 no? not 1 and 0?

* The ``subsection VOF`` defines the areas where both fluids lay at the initial state (see section :doc:`multiphysics`). In this example, the ``Function expression`` is used with a boolean condition to establish the region where the fluid indicator is 0 or 1: ``if (condition, value if true, value if false)``. ``if (x<0.5 & y<1, 1, 0)`` means that ``fluid 1`` initially fills the surface where ``x<0.5`` and ``y<1``, the rest being filled with ``fluid 0``.

> * The ``subsection cahn hilliard`` defines the areas where both fluids lay at the initial state (see section :doc:`multiphysics`). It works similarly to the ``subsection VOF`` for the first component, which corresponds to the phase order parameter. The user also has he choice to specify initial conditions for the chemical potential, though it is often better to leave it at 0.

Reviewer 2: ```suggestion
* The ``subsection cahn hilliard`` defines the areas where both fluids lay at the initial state (see section :doc:`multiphysics`). It works similarly to the ``subsection VOF`` for the first component, which corresponds to the phase order parameter. The user also has the choice to specify initial conditions for the chemical potential, although it is often more suitable to leave it at 0.
```

> * The ``subsection cahn hilliard`` defines the areas where both fluids lay at the initial state (see section :doc:`multiphysics`). It works similarly to the ``subsection VOF`` for the first component, which corresponds to the phase order parameter. The user also has the choice to specify initial conditions for the chemical potential, although it is often more suitable to leave it at 0.

Reviewer 3: ```suggestion
* The ``subsection cahn hilliard`` defines the areas where both fluids lay at the initial state (see section :doc:`multiphysics`). It works similarly to the ``subsection VOF`` for the first component, which corresponds to the phase order parameter. The user also has the choice to specify an initial condition for the chemical potential, although it is often more suitable to leave it at 0.
```
